### DISCUSSION TITLE
[BP#111]  « Préférer le texte brut au HTML »

### DISCUSSION THREAD

**Discussion by rlemaire** [cnumr/best-practices#20] [Bonnes pratiques] [answered]
Même si la différence est grande (x12 d'après l'étude de Frédéric citée dans ECO), on reste sur de petites quantités de données. Je pense qu'il vaudrait mieux parler des pièces jointes et des images dans les emails.

**Comment by jonathandetrierIF** [comment]
Je suis d'accord sur le fait qu'il faille parler des PJ et des images dans les mails, mais je ne suis pas totalement d'accord avec la suppression de cette BP. L'intérêt de cette BP pour moi est double : d'abord on fait des économies de données transmises et stockées quand on rapporte cela sur la masse (plus de 300 milliards de mails sont envoyés chaque jour). Ce ne sont pas des grandes quantités de données, mais la différence reste grande. Ensuite, le plus gros intérêt pour moi de cette BP est pédagogique : les e-mails sont ce que les consommateurs de services numériques voient le plus (bien plus qu'un ++$i à la place d'un $i++ dans du code PhP). Les habituer à des mails en texte brut ou épurés peut sembler anodin pour certains, mais ça transmet tout de même un message. Ca permet aussi au lecteur de prendre conscience de la quantité de données que représente la mise en page et tout ce qui est autour du contenu lui-même. De mon point de vue, cela serait plus pertinent de modifier cette BP en "Faire preuve de sobriété dans la rédaction des mails" ou quelque chose comme cela, dans lequel on pourrait aussi bien faire allusion au texte brut qu'aux PJ et images.

**Comment by rlemaire** [comment]
Je ne pense pas que ce soit l'endroit pour la pédagogie. On propose des pratiques aux professionnels qui ont déjà intégré les grandes idées.
Il y a une différence sensible pour les utilisateurs entre un texte brut et un texte html, mais (hors pièces jointes) on reste de petits volumes de données. On risque donc une perte conséquente pour les utilisateurs avec un grain modéré. De plus on risque d'entretenir les idées reçues sur l'impact des emails, le focus sur les usages, etc.
Je suis ok pour un remplacement par quelque chose du style « Limiter la taille des emails envoyés » avec dans la description une emphase particulière pour l'envoi d'un grand nombre de messages à des fins de marketing.

**Reply by jonathandetrierIF** [reply]
Effectivement, je n’avais pas pensé aux idées reçues que l’on pourrait entretenir de cette manière … 
« limiter la taille des e-mails envoyés » me semble être pertinent en effet.

**Reply by ldesgrange** [reply]
À voir si d'autres études vont dans le même sens, mais d'un point de vue "marketing", les e-mails en texte brut semblent avoir un taux d'ouverture plus important… https://blog.hubspot.com/marketing/plain-text-vs-html-emails-data

**Comment by ACTLEM** [comment]
Note équipe projet: Modification pour énoncer les pièces jointes: "Limiter la taille des e-mails envoyés"
Impact à réévaluer (moindre qu'actuellement), Opt-in a plus d'impact.

**Reply by ddebarbieux** [reply]
Je pourrais aller plus loin : limiter le nombre d'émails!

Dans les processus de commande en lignes, il n'est pas rare de recevoir deux mails en moins de 30 secondes.

**Reply by Christophe-Cl** [reply]
Pour aller encore plus loin : "Limiter les notifications" voir "Laisser l'utilisateur décider du type et du nombre de notifications souhaitées"

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/145

**Comment by Christophe-Cl** [comment]
Au delà de la taille des emails il y a toute une réflexion autour des usages mails : limiter les envois, les destinataires, les PJ, etc.
Une BP plus générale intégrant ces facteurs ?

**Comment by florinesueur** [comment]
En fat cette BP me pose question parce que je vois deux utilisations distinctes :

- Particuliers et employés qui envoient un mail à un ou un petit nombre de destinataires avec ou sans pièces jointes
- Professionnel de l'emailing de masse
- Notification système (reset password, nouveau message, validation de compte, ...)

Les autres BP directement en lien avec l'emailing sont :

- BP#109 - [Dédoublonner systématiquement les fichiers avant envoi](https://github.com/cnumr/best-practices/blob/main/chapters/BP_109_fr.md)
- BP#110 - [N'utiliser que des fichiers double opt-in](https://github.com/cnumr/best-practices/blob/main/chapters/BP_110_fr.md)

Comme dit par @rlemaire, nous nous adressons à des professionnels du web dont, personnellement, je ciblerais les webmarketeurs et intégrateurs mail.

Donc pour moi, il fait sens de :
- préférer le text brut au html ou l'utilisation de framework tel que MJML
- limiter l'utilisation de médias (images, images animées, vidéos)
- éviter à l'utilisateur de devoir aller sur le site en donnant toutes les informations dans le mail (exemple : linkedin vous avez un nouveau message sans la teneur du message en soi poussant à aller sur la plateforme)
- permettre à l'utilisateur de customiser et de se désinscrire : d'ajuster la périodicité d'envoi et de sélectionner les sujets qui l'interesse
- nettoyer ses contacts d'envoi (doublons, désinscriptions, non distribuées ...)
- mettre en place une stratégie de désabonnement automatique (un contact qui n'ouvre pas x emails ni l'email d'avertissement est désinscrit automatiquement, par exemple)
- N'utiliser que des fichiers double opt-in

**Reply by ddebarbieux** [reply]
Autrement dit, il y a trois bonnes pratiques sur les mails. Cela paraît beaucoup ? on pourrait fusionner dans une seule ?
